servers/physics_3d/godot_area_pair_3d.cpp

```diff
@@ -104,7 +104,8 @@ GodotAreaPair3D::GodotAreaPair3D(GodotBody3D *p_body, int p_body_shape, GodotAre
 
 GodotAreaPair3D::~GodotAreaPair3D() {
 	if (colliding) {
-		if (has_space_override) {
+		PhysicsServer3D::AreaSpaceOverrideMode area_gravity_mode = (PhysicsServer3D::AreaSpaceOverrideMode)(int)area->get_param(PhysicsServer3D::AREA_PARAM_GRAVITY_OVERRIDE_MODE);
+		if (has_space_override || area_gravity_mode != PhysicsServer3D::AREA_SPACE_OVERRIDE_DISABLED) {
```

Comment:
```suggestion
		if (has_space_override || (int)area->get_param(PhysicsServer3D::AREA_PARAM_GRAVITY_OVERRIDE_MODE) != PhysicsServer3D::AREA_SPACE_OVERRIDE_DISABLED) {
```
No need to keep a separate variable, further this avoids doing the fetch if `has_space_override` is true

Reply:
I've fixed that in the last push, thanks for the feedback!